src/Utilities/Constants.js

```diff
@@ -1,3 +1,2 @@
-
-export const SOURCES_API_BASE = `/r/insights/platform/topological-inventory/v0.0`;
-export const SERVICE_PORTAL_API_BASE = `/r/insights/platform/service-portal/v0.0`;
+export const SERVICE_PORTAL_API_BASE = process.env.BASE_PATH + '/service-portal/v0.0';
```

Comment:
Can you please use template strings. Its just about code styling.
```javascript
export const SERVICE_PORTAL_API_BASE = `${process.env.BASE_PATH}/service-portal/v0.0`;
```

Reply:
I really don't understand template strings.  Can you explain why they are preferable over normal strings or give me a link with a good explanation?

Reply:
@Fryguy for this case its just a different syntax. But it has some neat features especially for string interpolation. Also the `+` is ambiguous and in some cases may have bad side effects (if you don't have 100% control over the variable content). That won't happen in this case obviously but we are using it across the code and i would like to keep it consistent.

There is a lot of articles across the internet. But this one is good imo: https://developers.google.com/web/updates/2015/01/ES6-Template-Strings